Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -1101,12 +1103,36 @@ AccessibilityButtonState AccessibilityNodeObject::checkboxOrRadioValue() const
 }
 
 #if ENABLE(AX_THREAD_TEXT_APIS)
-bool AccessibilityNodeObject::shouldEmitNewlinesBeforeAndAfterNode() const
+TextEmissionBehavior AccessibilityNodeObject::emitTextAfter() const
```

Comment:
Nit: the name of this reads like a bool to me, but returns a `TextEmissionBehavior`. What do you think about calling this `textEmissionBehavior()`, or `emitTextAfterBehavior()`?

Reply:
Good point, `emitTextAfterBehavior` sounds better, will go with that! I anticipate we'll eventually need `emitTextBeforeBehavior`, hence why `textEmissionBehavior` may not work as well.